src/silx/gui/hdf5/Hdf5TreeModel.py

```diff
@@ -703,7 +709,7 @@ def insertFile(self, filename, row=-1):
             h5file = silx_io.open(filename)
             if self.__ownFiles:
                 self.__openedFiles.append(h5file)
-            self.sigH5pyObjectLoaded.emit(h5file)
+            self.sigH5pyObjectLoaded.emit(h5file, filename)
             self.insertH5pyObject(h5file, row=row, filename=filename)
```

Comment:
Not sure here why the signal is emitted **before** inserting the item into the model? :shrug: